.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: "Continuous Integration"
+on:
+  push:
+    # run on pushes to master or release/*
+    branches:    
+      - master
+      - release/*
+  pull_request:
+    # run on pull requests targeting master
+    branches:
+      - master
+jobs:
+  run-tests:
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Cluster
+      run: |
+        kind delete cluster 2>/dev/null
```

Comment:
you can do `|| true` if you don't want it to fail. also consider adding set +x; so that you can tell what's happening